viewers/nerf_viewer/client/src/themes/theme.ts

```diff
@@ -0,0 +1,180 @@
+import { createTheme } from '@mui/material/styles';
+
+export const appTheme = createTheme({
+  palette: {
+    primary: { main: '#EEEEEE' },
+    secondary: { main: '#FFD369' },
+    text: {
+      primary: '#EEEEEE',
+      secondary: '#FFD369',
+      disabled: '#555555',
+    },
+  },
+  components: {
+    MuiPaper: {
+      styleOverrides: {
+        root: {
+          backgroundColor: '#222831',
+        },
+      },
+    },
+    // MuiIcon: {
```

Comment:
If these lines does not work, and it is no important comment, please remove them from official commit. To keep them, push another branch in a forked repo.

Reply:
Unused code blocks removed.